openreview/conference/builder.py

```diff
@@ -164,9 +192,58 @@ def close_submissions(self, freeze_submissions = True):
 
         return invitation
 
+    def create_blind_submissions(self, public = False):
+
+        if not self.double_blind:
+            raise openreview.OpenReviewException('Conference is not double blind')
+
+        if next(self.get_submissions(), None):
+            raise openreview.OpenReviewException('Blind submissions already created')
+
+        self.invitation_builder.set_blind_submission_invitation(self)
+        blinded_notes = []
+
+        for note in tools.iterget_notes(self.client, invitation = self.get_submission_id()):
+            blind_note = openreview.Note(
+                original= note.id,
+                invitation= self.get_blind_submission_id(),
+                forum=None,
+                signatures= [self.id],
+                writers= [self.id],
+                readers= [self.id],
+                content= {
+                    "authors": ['Anonymous'],
+                    "authorids": [self.id],
+                    "_bibtex": None
+                })
+
+            posted_blind_note = self.client.post_note(blind_note)
```

Comment:
Why is the note posted twice?

Reply:
'number' is generated when it is posted the first time, using which we populate the correct per-paper author, reviewer, and area chair groups (as can be seen at line 227) which get added as readers for the blind note. Subsequent post is needed to save these changes.

Reply:
In there future we need to figure out a way to this without posting twice. May be we can pass the number value to the backend.